lib/Horizon.dart

```diff
@@ -0,0 +1,224 @@
+import 'package:flutter/material.dart';
+
+class Horizon extends StatelessWidget {
+    const Horizon({super.key});
+
+    @override
+    Widget build(BuildContext context) {
+        return  CustomScrollView(
+          slivers: <Widget>[
+            SliverAppBar(
+              title: Text('Horizons'),
+              backgroundColor: Colors.teal[500],
+              pinned: true,
+              expandedHeight: 200,
+              flexibleSpace: FlexibleSpaceBar(
+                title: const Text('Horizons'),
+                background: DecoratedBox(
+                  position: DecorationPosition.foreground,
+                  decoration: BoxDecoration(
+                    gradient: LinearGradient(
+                      begin: Alignment.bottomCenter,
+                      end: Alignment.center,
+                      colors: <Color>[Colors.teal[800]!, Colors.transparent],
+                    ),
+                  ),
+                  child: Image.network(
+                    headerImage,
+                    fit: BoxFit.cover,
+                  ),
+                ),
+              ),
+            ),
+            WeatherForecast(),
+          ],
+        );
+    }
+}
+
+class WeatherForecast extends StatelessWidget {
+    const WeatherForecast({super.key});
+
+    @override
+    Widget build(BuildContext context) {
+        final DateTime currentDate = DateTime.now();
+        final TextTheme textTheme = Theme.of(context).textTheme;
+        
+
+        print("currentDate , $currentDate");
+        print("currentDate.day, $currentDate.day");
+        print("currentDate.weekday, $currentDate.weekday" );
```

Comment:
Comment: In production code, these prints would have to be removed. I think you should remove them.